resources/migrations/20170219091344-shooters-suggest-trigger.up.sql

```diff
@@ -0,0 +1,24 @@
+--Add suggest column to shooters
+ALTER TABLE shooters
+  ADD COLUMN suggest TEXT,
+  ADD COLUMN updated_at TIMESTAMPTZ DEFAULT now();
```

Comment:
also do I dismiss the review? Is there anything I do to notify reviewers that a review has been addressed?